Gemfile.lock

```diff
@@ -170,6 +176,7 @@ GEM
     websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.4)
```

Comment:
Name: loofah
Version: 2.2.3
Advisory: CVE-2019-15587
Criticality: Unknown
URL: https://github.com/flavorjones/loofah/issues/171
Title: Loofah XSS Vulnerability
Solution: Upgrade to >= 2.3.1.